### ISSUE TITLE
List view

### ISSUE THREAD

**Issue by vismagine** [owncloud/calendar#85]
I am not able to find a simple list view for the calendar in ownCloud - is there any?

A simple list view of all events would be a great enhancement for many people using the calendar app as a time planer tool. Paired with an optional simple Export of the times + appointment names (no must have!) to a text based file format like CSV would make a time management tool out of the ownCloud calendar.

Sorry for posting this as an Issue rather than post it to the feature request list, which I was not able to find.

**Comment by georgehrke** [comment]
cc @jancborchardt @raghunayyar 

I think this csv feature is a complete overkill and not something we will do. A list view is something we could definitely consider.

**Comment by vismagine** [comment]
Yes, the CSV Export is definately not a must have. In my case it would enable me to sum up durations of events/appointments i had during a month, but this is certainly a special case.

Thanks a lot for considering the list view feature as a possibly enhancement for the calendar. Regarding the content of a list view it would be usefull to have fields in a row like:
* fromtime
* duration (in hours)
* event-title
* event-description (just as a button to be able to copy the whole text)

**Comment by zinks-** [comment]
Being able to chose what columns to show would be great!

**Comment by Mannshoch** [comment]
How easy could such a Listview be themable.

I need something like:
![jahreskalender](https://cloud.githubusercontent.com/assets/7795891/15067034/d2a5b816-136d-11e6-928b-18ac3faa773e.png)

**Comment by vacy** [comment]
I would like to see a solution like explained here: https://www.drupal.org/project/agenda
Any plans to develope that?

**Comment by georgehrke** [comment]
Fullcalendar 3 implements a new list view. :)
https://fullcalendar.io/blog/2016/09/feature-packed-releases/

So the list view will probably be part of Calendar 1.4

**Comment by georgehrke** [comment]
Fullcalendar 3 implements a new list view. :)
https://fullcalendar.io/blog/2016/09/feature-packed-releases/

So the list view will probably be part of Calendar 1.4

**Comment by georgehrke** [comment]
Fullcalendar 3 implements a new list view. :)
https://fullcalendar.io/blog/2016/09/feature-packed-releases/

So the list view will probably be part of Calendar 1.4

**Comment by georgehrke** [comment]
The current list view in fullcalendar seems to be a bit buggy. Some days are not summarized correctly, it's not chronological and it doesn't have continuous scrolling. :/

**Comment by tcitworld** [comment]
In FullCalendar or in this app with the new fullcalendar version ?

**Comment by georgehrke** [comment]
With the new fullcalendar version in the calendar app.
I added the listMonth view to the button area and observed the issues mentioned above
https://fullcalendar.io/docs/list_view/intro/
